ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/middleware/proxy_redirect.clj

```diff
@@ -41,9 +42,23 @@
       (.read rdr buf)
       buf)))
 
-(defn- slurp-body-binary
+(defn- not-delete?
   [request]
-  (if-let [len (when (:body request) (get-in request [:headers "content-length"]))]
+  (not= :delete (:request-method request)))
+
+(defn- body?
+  [request]
+  (:body request))
```

Comment:
My 2 cents.  The two expressions `(:body request)` and `(body? request)` seem equally clear and for all intents and purposes have the same effect.  Is `body?` needed at all?